scm-ui/build.gradle

```diff
@@ -84,11 +84,10 @@ task test(type: YarnTask) {
     include 'ui-*/**/*.ts'
     include 'ui-*/**/*.tsx'
   })
-  .withPathSensitivity(PathSensitivity.RELATIVE)
+    .withPathSensitivity(PathSensitivity.RELATIVE)
   outputs.dir('build/jest-reports')
 
   if (project.isCI) {
-    ignoreExitValue = project.isCI
```

Comment:
This is the actual change in the gradle task. Exit codes are not longer ignored by Jenkins but handled correctly in the build.